migrations/versions/fd4d749a0aa4_add_mission_uuid.py

```diff
@@ -0,0 +1,45 @@
+"""add mission uuid
+
+Revision ID: fd4d749a0aa4
+Revises: 6f32f51871d2
+Create Date: 2019-09-29 01:19:20.316651
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fd4d749a0aa4'
+down_revision = '6f32f51871d2'
+branch_labels = None
+depends_on = None
+
+# make sure this migration will keep working *magic*
+from spaceship.db import db
+db.Model.metadata.bind = op.get_bind()
+
+# imports for the data part of the migration
+from spaceship.models import Mission
+import uuid
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('mission', sa.Column('uuid', sa.String(length=32), nullable=True))
```

Comment:
It might be clearer to call this mission.mission_uuid or something since the types of things are not really clear in this orm thing and a priori I don't know what a mission.mission would be.